### ISSUE TITLE
Using SfePy in pycharm 

### ISSUE THREAD

**Issue by imselva** [sfepy/sfepy#627]
I am new to SfePy and I want to use it in pycharm. 
I downloaded the SfePy repository form Github and used Python3.7 as an interpreter in Pycharm.
When I run the Primer example in the SfePy documentation, I am getting the following error.

(base) C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master>python ./simple.py ./its2D_1.py
Traceback (most recent call last):
  File "./simple.py", line 21, in <module>
    from sfepy.applications import PDESolverApp, EVPSolverApp
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\applications\__init__.py", line 3, in <module>
    from .pde_solver_app import PDESolverApp, solve_pde, assign_standard_hooks
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\applications\pde_solver_app.py", line 7, in <module>
    from sfepy.discrete import Problem
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\discrete\__init__.py", line 9, in <module>
    from .variables import (Variables, Variable, FieldVariable, DGFieldVariable,
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\discrete\variables.py", line 21, in <module>
    from sfepy.discrete.fem.lcbc_operators import LCBCOperators
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\discrete\fem\__init__.py", line 11, in <module>
    from .fields_base import Field
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\discrete\fem\fields_base.py", line 26, in <module>
    from sfepy.discrete.fem.mappings import VolumeMapping, SurfaceMapping
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\discrete\fem\mappings.py", line 10, in <module>
    from sfepy.discrete.common.extmods.mappings import CMapping
ModuleNotFoundError: No module named 'sfepy.discrete.common.extmods.mappings'

ModuleNotFoundError: No module named 'sfepy.discrete.common.extmods.mappings'

How to solve this error?

**Comment by zitkat** [comment]
Hi, I have been developing for SfePy in PyCharm for past year. PyCharm is overall great IDE expirience. Your error is probably not related to using IDE but rather to building SfePy from source, see [here ](http://sfepy.org/doc-devel/installation.html#installing-sfepy-from-sources). It seems you need to build C extensions, running `python setup.py build_ext --inplace` from command line in your `sfepy-master` folder should do.

**Comment by imselva** [comment]
> Hi, I have been developing for SfePy in PyCharm for past year. PyCharm is overall great IDE expirience. Your error is probably not related to using IDE but rather to building SfePy from source, see [here ](http://sfepy.org/doc-devel/installation.html#installing-sfepy-from-sources). It seems you need to build C extensions, running `python setup.py build_ext --inplace` from command line in your `sfepy-master` folder should do.

Thank you so much for your response. That did help solve the first issue. 
But now I have another issue during post processing.

(base) C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master>python ./postproc.py its2D.vtk -b
Traceback (most recent call last):
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\postprocess\viewer.py", line 10, in <module>
    from enthought.traits.api \
ModuleNotFoundError: No module named 'enthought'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./postproc.py", line 53, in <module>
    from sfepy.postprocess.viewer import (Viewer, get_data_ranges,
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\sfepy\postprocess\viewer.py", line 27, in <module>
    from tvtk.pyface.scene_editor import SceneEditor
ModuleNotFoundError: No module named 'tvtk'

I have installed Mayavi in the same sfepy-master folder, but somehow the dependencies are missing I guess, 
https://pypi.org/project/mayavi/ I have tried everything in this link, still I am unable to run the postprocessing.

How to solve this error? 

(base) C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master>python setup.py build_ext --inplace
Traceback (most recent call last):
  File "setup.py", line 112, in _mayavi_version
    from enthought.mayavi import version
ModuleNotFoundError: No module named 'enthought'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 243, in <module>
    check_versions()
  File "setup.py", line 164, in check_versions
    show_only=show_only)
  File "C:\Users\Selva Prakash\Downloads\sfepy-master\sfepy-master\build_helpers.py", line 325, in package_check
    have_version = version_getter(pkg_name)
  File "setup.py", line 115, in _mayavi_version
    from mayavi import version
ImportError: cannot import name 'version' from 'mayavi' (unknown location)

Where should I keep the mayavi folder??
